tests/codex/testerasure.nim

```diff
@@ -252,3 +252,22 @@ suite "Erasure encode/decode":
       decoded.treeCid == manifest.treeCid
       decoded.treeCid == verifiable.originalTreeCid
       decoded.blocksCount == verifiable.originalBlocksCount
+
+  for i in 1..5:
```

Comment:
I'm not exactly a fan of random in unit tests. Perhaps we can nest a few for-loops and test several combinations?
```
  for datasetSize in 1..x:
    for ecK in y..x:
```
etc
If nothing else, it'd be cool to see which points in this space are giving us trouble.

Reply:
I think that fuzzy testing is a pragmatic approach to the problems with software failing on unexpected input (case that we do observe).

Of course there's more to it, like we should print rng seed whenever a failure is detected so that an error can be reproduced deterministically. Unfortunately this require more library kind of work to get it working properly.

Reply:
I think both "golden" test and randomized tests are important. But maybe they should be separated into two categories? (I have very good experience with randomized testing)